client/templates/globals/resetPassword.js

```diff
@@ -0,0 +1,17 @@
+import { Template } from 'meteor/templating';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+
+Template.resetPassword.events({
+    'submit #at-pwd-form': function (event) {
+        event.preventDefault();
+        let token = FlowRouter.getParam("token");
+        Accounts.resetPassword(token, $('#at-field-password').val(), ( error ) =>{
+            if ( error ) {
+                alert(error.reason);
```

Comment:
I would prefer the FlashMessage instead of a simple alert. The same applies for the case of success in the else-branch.